### PR TITLE
Update Privacy Sandbox OT feature detection

### PR DESCRIPTION
**Pull request by kevinkiklee**
Add Permissions Policy feature detection code. Also add a separate entry for `joinAdInterestGroup()`.

### CODE REVIEW

**Review by samdutton — APPROVED** [review]
LGTM.

Not sure if "browsing-topics" should be `browsing-topics` (etc.)

File: site/en/blog/privacy-sandbox-unified-origin-trial/index.md [hunk 1]
```diff
@@ -200,31 +200,38 @@ before using it.
...
+```javascript
+if ('runAdAuction' in navigator && document.featurePolicy.allowsFeature('run-ad-auction')) {
+  // FLEDGE auction enabled
+}
+```
 
 ### Attribution Reporting
 
-Check for the `attributionReporting` object in the `window`.
+Check for the `attributionReporting` object in the `window`, and the permissions policy for "attribution-reporting".
```
Comment (heyawhite) [inline site/en/blog/privacy-sandbox-unified-origin-trial/index.md:231]:
```suggestion
Check for the `attributionReporting` object in the `window` and
the Permissions Policy for "attribution-reporting".
```

File: site/en/blog/privacy-sandbox-unified-origin-trial/index.md [hunk 2]
```diff
@@ -200,31 +200,38 @@ before using it.
...
-if ('browsingTopics' in document) {
+if ('browsingTopics' in document && document.featurePolicy.allowsFeature('browsing-topics')) {
   // Topics enabled
 }
 ```
 
 ### FLEDGE
 
-Check for the `runAdAuction`function  in `navigator`. 
+If you want to join an ad interest group, check for the `joinAdInterestGroup` function in `navigator`, and the permissions policy for "join-ad-interest-group".
```
Comment (heyawhite) [inline site/en/blog/privacy-sandbox-unified-origin-trial/index.md:213]:
```suggestion
If you want to join an ad interest group, check for the `joinAdInterestGroup` function in
`navigator` and the Permissions Policy for "join-ad-interest-group".
```

File: site/en/blog/privacy-sandbox-unified-origin-trial/index.md [hunk 3]
```diff
@@ -200,31 +200,38 @@ before using it.
 
 ### Topics
 
-Check for the `browsingTopics()` function in the `document`.
+Check for the `browsingTopics()` function in the `document`, and the [permissions policy](/docs/privacy-sandbox/permissions-policy/#featurepolicyallowsfeaturefeature) for "browsing-topics".
```
Comment (heyawhite) [inline site/en/blog/privacy-sandbox-unified-origin-trial/index.md:203]:
```suggestion
Check for the `browsingTopics()` function in the `document` and
the Permissions Policy](/docs/privacy-sandbox/permissions-policy/#featurepolicyallowsfeaturefeature)
for "browsing-topics".
```

**Review by heyawhite — APPROVED** [review]

**Comment by heyawhite** [comment]
@kevinkiklee to merge at your leisure

**Pull request merged**